src/clickup/__tests__/e2e.clickup.test.ts

```diff
@@ -0,0 +1,129 @@
+
+      expect(Object.keys(projects[0])).toEqual(expect.arrayContaining([
+        'id',
+        'remote_id',
+        'name',
+        'description',
+        'created_at',
+        'modified_at',
+      ]));
+
+      expect(projects[0]).toEqual({
+        id: expect.stringMatching(ID_REGEX),
+        remote_id: expect.any(String),
+        name: expect.any(String),
+        description: expect.any(String),
+        created_at: expect.any(Number),
+        modified_at: expect.any(Number),
+      });
+    });
+
+    it('getProjects failed with "Team IDs not found" error', async () => {
+      await expect(service.getProjects()).rejects.toThrow('Team IDs not found');
+    });
+
+  });
+
+  describe('getTasks', () => {
+    it('getTasks succeeded', async () => {
+      await service.getUsers();
+      await service.getProjects();
+      const tasks: IMerjoonTasks = await service.getTasks();
+
+      expect(Object.keys(tasks[0])).toEqual(expect.arrayContaining([
+        'id',
+        'remote_id',
+        'name',
+        'assignees',
+        'status',
+        'description',
+        'projects',
+        'remote_created_at',
+        'remote_modified_at',
+        'created_at',
+        'modified_at',
+      ]));
+
+      expect(tasks[0]).toEqual({
+        id: expect.stringMatching(ID_REGEX),
+        remote_id: expect.any(String),
+        name: expect.any(String),
+        assignees: expect.arrayContaining([expect.stringMatching(ID_REGEX)]),
+        status: expect.any(String),
+        description: expect.any(String),
+        projects: expect.arrayContaining([expect.stringMatching(ID_REGEX)]),
+        remote_created_at: expect.any(String),
+        remote_modified_at: expect.any(String),
+        created_at: expect.any(Number),
+        modified_at: expect.any(Number),
+      });
+    });
+
+    it('getTasks failed with "List IDs not found" error', async () => {
+      await expect(service.getTasks()).rejects.toThrow('List IDs not found');
+    });
+
+    it('getTasks failed with "List IDs not found" error', async () => {
+      await service.getUsers();
+      await expect(service.getTasks()).rejects.toThrow('List IDs not found');
+    });
+  });
+
+  describe('Check References', () => {
+    it('checkReferences', async () => {
+      const users: IMerjoonUsers = await service.getUsers();
+      const projects: IMerjoonProjects = await service.getProjects();
+      const tasks: IMerjoonTasks = await service.getTasks();
+  
+      for (const task of tasks) {
+        const assigneeIds = task.assignees.map((assignee) => assignee);
+        const userIds = users.map((user) => user.id);
+        expect(userIds).toEqual(expect.arrayContaining(assigneeIds));
+  
+        const taskProjectIds = task.projects.map((project) => project);
+        const projectIds = projects.map((proj) => proj.id);
+        expect(projectIds).toEqual(expect.arrayContaining(taskProjectIds));
+      }
+    }, 70000);
```

Comment:
I think the default 5 sec timeout should be enough to test this out. oN sure why you need 70 secs here